src/main/java/com/google/devtools/build/lib/bazel/bzlmod/ModuleFileFunction.java

```diff
@@ -445,13 +445,14 @@ public static RootModuleFileValue evaluateRootModuleFile(
       throw errorf(Code.BAD_MODULE, "error executing MODULE.bazel file for the root module");
     }
     for (ModuleExtensionUsage usage : module.getExtensionUsages()) {
-      if (usage.getIsolationKey().isPresent() && usage.getImports().isEmpty()) {
+      ModuleExtensionUsage.Proxy firstProxy = usage.getProxies().getFirst();
```

Comment:
Maybe move this into the `if` and rename to `onlyProxy`?

Reply:
the `if` itself contains a reference to `firstProxy`.